.azure-pipelines/templates/github-release-stage/version-and-release.yml

```diff
@@ -0,0 +1,19 @@
+steps: 
+- task: PowerShell@2
+  inputs:
+    filePath: '$(System.DefaultWorkingDirectory)/_msgraph-sdk-java-core build and package/drop/Scripts/getLatestVersion.ps1'
+    arguments: '-propertiesPath "$(PROPERTIES_PATH)"'
+
+- task: GitHubRelease@1
+  inputs:
+    gitHubConnection: 'ramsessanchez'
```

Comment:
maybe use a template parameter for that value? so we can surface the parameter all the way up to the pipeline definition instead of "being hidden down here"

Reply:
Just to check, does the GitHubRelease task still only accept an OAuth token based connection? Ideally it would the Azure Pipelines app connection named `microsoftgraph`.

Reply:
according to [the documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/github-release?view=azure-devops) a service connection should be fine so long as it has the proper permissions on the target repo.